### ISSUE TITLE
Remove wasmycartowed.com - no longer the open source project

### ISSUE THREAD

**Issue by geneorama** [chihacknight/chihacknight.org#262]
On the list of open source projects here: https://chihacknight.org/open-source-projects.html

I don't think http://wasmycartowed.com/ is still the open source Scott Robbins project.

**Comment by derekeder** [comment]
Yikes you're right! 

Nothing lasts forever ... not even was my car towed dot com *guitar solo*

**Comment by derekeder** [comment]
hmm, well the repo is still open at https://github.com/srobbin/wasmycartowed

I'm inclined to leave it for posterity. Though the URL listed in the repo is def going to a different place. @srobbin would you mind updating the URL on your repo?

**Comment by geneorama** [comment]
Yes, the repository is fine.

I think someone let the URL ownership lapse and someone pounced on it.

I don’t think we want to drive traffic to the new owner.

It’s certainly up to you!

**Comment by srobbin** [comment]
@derekeder This whole thing is a bummer. I transferred the app and website into the care of Smart Chicago, but I think it got lost in the shuffle of the transition to City Tech and the domain wasn't renewed. Technically, the app is [still running on Heroku](http://wasmycartowed.herokuapp.com/)—it just doesn't have the good URL anymore.

In any case, I just removed the URL from the repo description, so there's no confusion.
